### PR TITLE
Validate BRUH

### PR DESCRIPTION
**Pull request by bigdenergy**
# Validate [BRUH](https://solscan.io/token/GfqwTaqqzPkH6xzbYZyS9fXku34Zj3kJ47n42PjKeZBf)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://x.com/bruhwifcap/status/1749019815824540029?s=20)
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/{id}

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by bigdenergy** [comment]
BRUH VALIDATION

**Comment by flamula** [comment]
Validatooors.

[$BRUH](https://twitter.com/search?q=%24BRUH&src=cashtag_click) CA:GfqwTaqqzPkH6xzbYZyS9fXku34Zj3kJ47n42PjKeZBf
https://x.com/bruhwifcap/status/1749851823685382258?s=20

wen meme token category sers! plz thx.

File: validated-tokens.csv [hunk 1]
```diff
@@ -1063,3 +1063,4 @@ PRE-TOKEN,PRE-,EZGvDFqrMSAxTt3Ud2bcguASaPKwGExiDzHVLqT3STZm,6,https://bafybeiceq
 BOZO,$BOZO,BoZoQQRAmYkr5iJhqo7DChAs7DPDwEZ5cv1vkYC9yzJG,5,https://arweave.net/QN7RFzy_OqPO_cAbBtBUzcUwObkDuvkRLS16oxJI-s0,true
 Pax Dollar,USDP,HVbpJAQGNpkgBaYBZQBR1t7yFdvaYVp2vCQQfKKEN4tM,6,https://424565.fs1.hubspotusercontent-na1.net/hubfs/424565/USDPLOGO.png,true
 Saros,SAROS,SarosY6Vscao718M4A778z4CGtvcwcGef5M9MEH1LGL,6,https://rapid.coin98.com/Currency/saros.png,true
+bruh wif cap,BRUH,GfqwTaqqzPkH6xzbYZyS9fXku34Zj3kJ47n42PjKeZBf,6,https://www.bruuuh.xyz/bruh_transparent.png,true
```
Comment (DegenDoes) [inline validated-tokens.csv:1066]: Looks good

**Review by DegenDoes — COMMENTED** [review]
Mint address: fqwTaqqzPkH6xzbYZyS9fXku34Zj3kJ47n42PjKeZBf
✅Symbol: BRUH
✅Token Decimals: 6
✅Token icon: https://www.bruuuh.xyz/bruh_transparent.png
✅On-chain metadata matches
✅No duplication on Jupiter's strict list

**Comment by DegenDoes** [comment]
Mint address: fqwTaqqzPkH6xzbYZyS9fXku34Zj3kJ47n42PjKeZBf
✅Symbol: BRUH
✅Token Decimals: 6
✅Token icon: https://www.bruuuh.xyz/bruh_transparent.png
✅On-chain metadata matches
✅No duplication on Jupiter's strict list

**Comment by DegenDoes** [comment]
> Validatooors.
> 
> [$BRUH](https://twitter.com/search?q=%24BRUH&src=cashtag_click) CA:GfqwTaqqzPkH6xzbYZyS9fXku34Zj3kJ47n42PjKeZBf https://x.com/bruhwifcap/status/1749851823685382258?s=20
> 
> wen meme token category sers! plz thx.

yes, confirm this. All good

**Comment by DegenDoes** [comment]
Coin Gecko added today - https://www.geckoterminal.com/solana/pools/ATNVk8KkKRyVaoxTzbDGC7DUaZQH5ePkyisY8j17eaa4?utm_source=coingecko&utm_medium=referral&utm_campaign=searchresults

**Comment by Jymfletzfur** [comment]
Insufficient mcap, volume

**Pull request closed without merging**